### DISCUSSION TITLE
New releases view hides release notes behind an expansion button

### DISCUSSION THREAD

**Discussion by abjugard** [community/community#6418] [Other Feature Feedback, Questions, & Ideas] [answered]
When I go to releases, I want to see the release, not the community reactions to the release. The releases, or at the very least the latest release, should be expanded to put the release notes front and center.

Also the new view has way too much padding, what's with all the space between the leftmost column of information, and the release notes column?

**Comment by MylesBorins** [comment, marked as answer]
Hey @abjugard, thanks for the feedback. We just shipped an update to minimize the space on the left column.

If you want to see the releases expanded by default you can add the following attribute to any release list page `&expanded=true`. This is included by default on any searches.
